tests/server/validation.js

```diff
@@ -0,0 +1,43 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const assert = intern.getPlugin('chai').assert;
+const { registerSuite } = intern.getInterface('object');
+const request = require('request-promise');
+const validation = require('../../server/lib/validation');
+
+const METRICS_DOCS_URL = 'https://raw.githubusercontent.com/mozilla/application-services/master/docs/accounts/metrics.md';
```

Comment:
Do we have other tests that go out on the internets to check stuff? 

Let's see how this works out in practice to see if anyone hits an "offline fail" in this case.